lib/cprofiles/include/cprofiles/cprof_info.h

```diff
@@ -20,14 +20,14 @@
 #ifndef CPROF_INFO_H
 #define CPROF_INFO_H
 
-#define CPROF_SOURCE_DIR "/src/fluent-bit"
+#define CPROF_SOURCE_DIR "/Users/james.leddy/src/fluent-bit"
```

Comment:
Something wrong here, `CPROF_SOURCE_DIR` should not be changed

Reply:
My mistake. I made this change to the wrong branch